pkg/osv/osv_test.go

```diff
@@ -0,0 +1,43 @@
+// pkg/osv/osv_test.go
+
+package osv
+
+import (
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Let's use the std libs for asserting rather than pull in a new package

Reply:
I've switched standard library assertions for simplicity. Let me know if there’s anything more to address. Thanks!